### ISSUE TITLE
installation from repository packages

### ISSUE THREAD

**Issue by sharethewisdom** [doronbehar/pistol#5]
I uploaded an [aur](https://aur.archlinux.org/packages/pistol-git/) package. Could you take a look?

Could other package managers be supported too?

**Comment by doronbehar** [comment]
Thanks for packaging Pistol @sharethewisdom! TBH, personally I don't have any motive to create packages of Pistol for any distro, not even my own (NixOS). As a developer of this program, I just install it to `$GOPATH/bin/` and this is how I continuously test it while changing it..

You do have a point though, that for the sake of other users, it might be worth suggesting distro specific packages for the ease of installation. I've opened the WiKi in the repo and I've mentioned your AUR package [here](https://github.com/doronbehar/pistol/wiki/GNU-Linux-Distributions'-Packages). I've also updated the README to mention that WiKi page.
